docs/user_documentation/ref-expressions.md

```diff
@@ -116,13 +116,11 @@ __Table B__
 ```js
 // Expressions are based on table A
 
-var cookies = []
 $('cookies').map(function (cookie) {
-    // You should use one of the following
-    cookies.push(cookie.value()) // results in ['1', '2', '3']
-    cookies.push(cookie.val.id) // results in ['1', '2', '3']
-    cookies.push(cookie.val.name) // results in ['Chocolate chip', 'Strawberry cookie', 'Banana cookie']
-    cookies.push(cookie.val.tastiness) // results in ['9/10', '10/10', '7/10']
-});
-cookies
+    // Pick one of the following
+    return cookie.value()           // results in ['1', '2', '3']
+    return cookie.value().id        // results in ['1', '2', '3']
+    return cookie.value().name      // results in ['Chocolate chip', 'Strawberry cookie', 'Banana cookie']
+    return cookie.value().tastiness // results in ['9/10', '10/10', '7/10']
+})
```

Comment:
add .value()

Reply:
This commit already sits in master, I think github is messed up.